L/LLVM/common.jl

```diff
@@ -366,11 +369,18 @@ ninja -j${nproc} -vv
 # Install!
 ninja install
 
-# Life is harsh on Windows and dynamic libraries are
-# expected to live alongside the binaries. So we have
-# to copy the *.dll from bin/ to tools/ as well...
-if [[ "${target}" == *mingw* ]]; then
-    cp ${prefix}/bin/*.dll ${prefix}/tools/
+if [[ "${LLVM_MAJ_VER}" -ge "16" ]]; then
+    #We can now tell cmake to put the dlls in the right place, and the verifier doesn't find them
+    if [[ "${target}" == *mingw* ]]; then
+        cp ${prefix}/tools/*.dll ${prefix}/bin/
+    fi
+else
+    # Life is harsh on Windows and dynamic libraries are
+    # expected to live alongside the binaries. So we have
+    # to copy the *.dll from bin/ to tools/ as well...
+    if [[ "${target}" == *mingw* ]]; then
+        cp ${prefix}/bin/*.dll ${prefix}/tools/
+    fi
```

Comment:
I prefer verbose copies, we've had a few issues with files being copied to the wrong place, with little means to debug the issue if nothing is written to the log file:
```suggestion
    #We can now tell cmake to put the dlls in the right place, and the verifier doesn't find them
    if [[ "${target}" == *mingw* ]]; then
        cp -v ${prefix}/tools/*.dll ${libdir}/.
    fi
else
    # Life is harsh on Windows and dynamic libraries are
    # expected to live alongside the binaries. So we have
    # to copy the *.dll from bin/ to tools/ as well...
    if [[ "${target}" == *mingw* ]]; then
        cp -v ${libdir}/*.dll ${prefix}/tools/.
    fi
```